### PR TITLE
assertDate

### PR DESCRIPTION
**Pull request by ilovemane**
I need this in omock

### CODE REVIEW

**Comment by ilovemane** [comment]
include                             na = FALSE,
                            null = FALSE,
                            unique = FALSE,
                            named = FALSE,

**Comment by ilovemane** [comment]
this should be fine now

File: R/assert.R [hunk 1]
```diff
@@ -557,6 +557,69 @@ assertTrue <- function(x,
...
+#' @param unique Whether it has to contain unique elements.
+#' @param named Whether it has to be named.
+#' @param call Call argument that will be passed to `cli` error message.
+#' @param msg Custom error message
+#'
+#' @return x
+#' @export
+#'
+assertDate <- function(x,
+                       length,
```
Comment (catalamarti) [inline R/assert.R:575]: can we have default of length = NULL, so any length it is allowed by default?

**Review by catalamarti — CHANGES_REQUESTED** [review]

File: R/assert.R (hunk 1 above)
Comment (ilovemane) [reply R/assert.R]: done

**Comment by catalamarti** [comment]
thanks @ilovemane

**Pull request merged**